### ISSUE TITLE
input helper removes list="mylist" attribute in rendered html (v1.11 and up) 

### ISSUE THREAD

**Issue by samheuck** [emberjs/ember.js#10908]
Here is a JSBin demonstrating the issue:
http://emberjs.jsbin.com/sabofi/2/edit?html,output

The input helper accepts attributes with static strings as variables - for instance:

```handlebars
{{input autocomplete="off"}}
```

In Ember v1.10.0

```handlebars
{{input list="mylist"}}
```

Will render

```html
<input type="text" list="mylist">
```

Where as ember v1.11.0 will render

```html
<input type="text">
```

I'd like to try to write a failing test for this - however I'm fairly new here and I'm still working out how to do such things. Thanks for taking a look.

**Comment by mixonic** [comment]
The change is that the value was set with `setAttribute` in 1.10, and in 1.11 is set with a property.

I cannot seem to recreate the exception on the console, though it is definitely raising from the app code itself. Confirmed in Chrome and FF.

**Comment by mixonic** [comment]
`list` is definitely not a read-only attribute: http://www.w3.org/TR/html5/forms.html#the-list-attribute

Unsure why the property would not be writable. The simplest short-term fix is to whitelist `list` as a `setAttribute` attribute on `input` elements.

**Comment by rwjblue** [comment]
Fixed on master, will be working in 1.13.0.

Update JSBin demo: http://emberjs.jsbin.com/yepadi/1/edit
